code/src/renderer/utils/help.js

```diff
@@ -1,5 +1,5 @@
 // eslint-disable-next-line import/prefer-default-export
 export function openHelp() {
   console.log("cat");
```

Comment:
is this console.log necessary?!?

Reply:
oh wow, I didn't notice this :P

I'll get rid of it!

Reply:
I'm surprised the linter didn't catch this, actually - I thought it was set to look for console.logs and complain about them